hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -363,6 +366,20 @@ public boolean delete(final Path f, final boolean recursive) throws IOException
 
   }
 
+  @Override
+  public ContentSummary getContentSummary(Path f) throws IOException {
+    org.apache.hadoop.fs.azurebfs.utils.ContentSummary contentSummary = null;
+    try {
+      contentSummary = (new ContentSummaryProcessor(abfsStore)).getContentSummary(f);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
Plase remove printstacktrace. Log the exception and rethrow as an IOException.

Reply:
done